docs/releases/1.9.txt

```diff
@@ -106,7 +106,10 @@ File Uploads
 Forms
 ^^^^^
 
-* ...
+* :class:`~django.forms.CharField` now accepts a
+  :attr:`~django.forms.CharField.strip` argument to strip input data of leading
+  and trailing whitespace.  As this defaults to ``True`` this is different
```

Comment:
Should be mentioned in backwards incompatible changes section of the release notes